pkg/cmd/gist/edit/edit.go

```diff
@@ -236,6 +236,8 @@ func editRun(opts *EditOptions) error {
 		if filename == "" {
 			if len(candidates) == 1 {
 				filename = candidates[0]
+			} else if len(candidates) == 0 {
+				return errors.New("No file found in this gist.")
```

Comment:
Please update the message to the suggestion below. It's because error messages should not be capitalised nor end with full stops.

```suggestion
				return errors.New("no file in the gist")
```